tests/test-pool-search.rb

```diff
@@ -226,6 +226,15 @@ def test_include(all,one)
     test_include(all_identifiers, first_identifier)
   end
 
+  it "#{url} should return at least one result for a * search" do
```

Comment:
This should be in the functional tests section.  The difference is that all the api tests should pass regardless of whether/what content is present in the index, and the functional tests can be used as a smoke test against a new version of the software to see if nothing major broke.